src/forms/helpers/dropdown-select.js

```diff
@@ -8,27 +8,38 @@ const propTypes = {
   children: PropTypes.node,
   className: PropTypes.string,
   selectedValues: PropTypes.arrayOf(PropTypes.string),
+  options: PropTypes.arrayOf(
+    PropTypes.shape({
+      key: PropTypes.string.isRequired,
+      value: PropTypes.string.isRequired,
+    })
+  ),
 }
 
 const defaultProps = {
   className: '',
   selectedValues: [],
+  options: [],
 }
 
-// Wraps the `DropdownCheckboxGroup` component
+// Wraps the `CheckboxGroup` component when dropdown is set to true.
 
-function DropdownSelect({ children, className, selectedValues }) {
+function DropdownSelect({ children, className, selectedValues, options }) {
   const [expanded, toggleExpanded] = useToggle()
 
   return (
     <OutsideClickHandler onOutsideClick={() => toggleExpanded(false)}>
-      <div className="dropdown-select">
+      <div
+        className={classnames('dropdown-select', {
+          'is-active': expanded,
```

Comment:
It seems to be a good idea to put "is-active" class name at the top level for styling purpose (e.g., coloring the border of the input when the dropdown menu is open)

Reply:
cc: https://github.com/LaunchPadLab/client-template/pull/456/files#diff-102f9f47138cbbdf4cafcec5b8d376a8a3e508d085c7a18668ca5704039fc207R271